api/src/main/resources/plugins/sparkStreaming.py

```diff
@@ -153,8 +148,7 @@ def create_component(self, staged_component_path, application_name, user_name, c
         logging.debug("uninstall commands: %s", undo_commands)
 
         start_commands = []
-        if usesSystemd:
-            start_commands.append('sudo systemctl daemon-reload\n')
+        start_commands.append('sudo systemctl daemon-reload\n')
         start_commands.append('sudo service %s start\n' % service_name)
```

Comment:
what about using systemctl instead of service as we are now using only RHEL/CentOS?

Reply:
Just that there's no need to change it, changes can always cause problems :-). 'service' is OK yes?